src/extension.ts

```diff
@@ -111,6 +111,12 @@ export async function activate(context: vscode.ExtensionContext) {
 
 	// create the view
 	createIntegrationsView();
+
+	// open at startup if setting is true
+	const openAtStartup : boolean = getOpenAtStartupSetting();
+	if (openAtStartup === true) {
```

Comment:
I think `if (openAtStartup)` would be a better check because it covers both the undefined and the true/false case. Though it seems that `getOpenAtStartupSetting()` can only return a boolean.